### ISSUE TITLE
Joystick Calibration Control Panel locks up if both buttons pressed simultaneously

### ISSUE THREAD

**Issue by thellmer-source** [a2stuff/a2d#346]
**Describe the bug**
Installed 1.2 preAlpha 23 today and was playing around with the various control panels and noticed in the Joystick panel as I was pressing the joystick buttons I accidentally pressed both buttons at once and the whole app locked up and became unresponsive.  I was able to click the close-square (which had turned black) and the panel closed, but then was repeatedly able to reproduce every time.  I have a CH 2 square button one center fire button joystick and it is the two square buttons when pressed at the same time will lockup the panel app.

**To Reproduce**
1. Start DeskTop
2. From the Apple menu, select 'Control Panels'
3. Go to 'Joystick Control Panel'
4. Press the two regular joystick buttons at once
6. App locks up

**Expected behavior**
I would expect the black button-depressed indicator dots to simply appear for both button indicators

**System Details (please complete the following information):**
 - System Apple IIe Enhanced, 128K 
 - Emulator?No - Using Real hardware
 - ProDOS version 2.4.2
 - DeskTop version 1.2 Alpha 23
 - Other  - 80 col/64-AUX, SSC - S1, Timemaster II - S2, Uthernet II - S3, Apple Mouse card - S4, 1MB /RAM5 Apple Extended Card - S5, 5.25 Unidisk floppy controller - S6 with single 5.25 Unidisk-type floppy attached, Rev C SCSI card - S7 - with Apple SC20 SCSI Hard drive attached.  Running A2D from Hard drive setup

**Additional context**
I never tried it on earlier versions of A2D...was an accident that I found it tonight when I pressed the buttons at once.  Could have been an existing issue from original 1.1...not sure.

**Comment by thellmer-source** [comment]
Im going to close this issue...completely reinstalled preAlpha 23 again and issue has disappeared - simultaneous button pressing behaves normally now.  Perhaps it was a corrupt ctrl panel file.

**Comment by inexorabletash** [comment]
It's also possible it wasn't a lockup, but you triggered [mouse keys mode](https://www.a2desktop.com/docs#h.p_fthAmDxtEOQ9) which has strange behavior.

**Comment by thellmer-source** [comment]
Yeah no telling.  It kept doing it after several reboots and power-downs, so I logged the error, thinking maybe I had run across a glitch since everything else seemed to be working fine. However, I had issues that whole afternoon with other ADTPro transfers as well - think something on the MBP or network was screwy as it kept giving host errors and timeouts on the IIe side which I usually never get.  So I rebooted the MBP, the bridge, and all my routers and APs then thought what the heck, let me try a fresh copy.  So after I copied it over again the issue no longer occurs.   So I suspect some file was corrupt maybe.
